### PR TITLE
To markdown options fix

### PR DESCRIPTION
**Pull request by anna-git**
Issue related : https://github.com/IonicaBizau/medium-editor-markdown/issues/26

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -25,6 +25,6 @@
   "dependencies": {
     "medium-editor": "^5.0.0",
     "to-markdown": "^2.0.1",
-    "uglify-js": "^2.5.0"
+    "UglifyJS2": "https://github.com/mishoo/UglifyJS2.git#^2.5.0"
```
Comment (IonicaBizau) [inline bower.json]: Can we move this in `devDependencies` maybe (if there are no `devDependencies` field, create it)?

Also, is there a specific reason why you chose to use the git url (:bulb: **Tip**: You can directly do `"mishoo/UglifyJS2#^2.5.0"`)? :grin:

File: DOCUMENTATION.md [hunk 2]
```diff
@@ -9,5 +9,6 @@ Creates a new instance of `MeMarkdown`.
 - **Object** `options`: An object containing the following fields:
  - `events` (Array): An array with the events when the markdown code will be generated (default: `["input", "change"]`).
  - `callback` (Function): The callback function. If the second argument is a function, then it has greater priority.
+ - `toMarkdownOptions` (Object): Options to pass to the markdown converter code.
```
Comment (IonicaBizau) [inline DOCUMENTATION.md:12]: Make sure to add this in the code as well. The `DOCUMENTATION.md` file is generated. :construction_worker:

**Comment by IonicaBizau** [comment]
Thanks a lot, @annayafi! Overall this looks amazing! :sparkles: 

There are few minor things to be fixed–I commented inline. :four_leaf_clover:

File: bower.json (hunk 1 above)
Comment (IonicaBizau) [reply bower.json]: :sparkles:

Comment (anna-git) [reply bower.json]: Without the url, simply "  "uglify-js": "^2.5.0"" wasn't working for me, didn't seem to find it on bower. Ok for devDependencies, I did the same for node, uglify and barbe went to devdependencies :)

Comment (IonicaBizau) [reply bower.json]: @annayafi Sounds good! :+1: The build process is a bit strange in this repository–probably we'll change in the future. I don't really like how I did that templating stuff and function stringifying etc. But it works. :grin:

**Pull request merged**